### DISCUSSION TITLE
MyDramaList

### DISCUSSION THREAD

**Discussion by dear-clouds** [catppuccin/catppuccin#1939] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the requested port be themed?

It's a simple userstyle :)

### Has the requested port been themed already?

I've created the port: https://github.com/dear-clouds/mydramalist

### Any additional comments?

Just le me know if I need to make any changes. Thanks :)

**Comment by sgoudham** [comment]
Hey @dear-clouds, thanks for creating the repository. 

In the time since this was created, we've collated all userstyles into one single location -> [catppuccin/userstyles](https://github.com/catppuccin/userstyles) with a defined set of standards for each userstyle to ensure consistency. 

I'm sure people would appreciate if this was refactored and added via a Pull Request to the userstyles repository, if you are no longer interested in picking this back up - I'm sure @isabelroses would be interested in taking the existing body of work and transferring it over. 

Apologies for not commenting earlier!

**Comment by dear-clouds** [comment]
Hi, thank you, I'll make a PR soon then :)

**Comment by WalkQuackBack** [comment]
Was a PR made for this? If not, I'm willing to refactor it and PR it to the repository.

**Reply by dear-clouds** [reply]
Omg i'm sorry, I completely forgot about this, please do!

**Comment by WalkQuackBack** [comment]
@dear-clouds Forgot to follow up back here, but made a PR here: https://github.com/catppuccin/userstyles/pull/2009
